helix-term/src/config.rs

```diff
@@ -20,6 +21,25 @@ pub struct LspConfig {
     pub display_messages: bool,
 }
 
+impl Config {
+    pub fn load(config_path: PathBuf) -> Result<Config, Error> {
+        match std::fs::read_to_string(config_path) {
+            Ok(config) => Result::Ok(toml::from_str(&config)
```

Comment:
Do we need to read from toml for config every time? Can't we just save the struct and reuse it rather than having to reparse it on every `load`?

Reply:
If the file changed then yes of course you need to re-read. The caching needs to happen on application level